### ISSUE TITLE
train_cifar.py TO our desired architecture

### ISSUE THREAD

**Issue by harleengulati03** [Deepkumar-Pawar/adl-coursework#11]
- [ ] Do we need to change the ImageShape class which currently has height, width and channels to include resolutions?
- [ ] In MAIN need to change train_dataset and test_dataset
- [ ] model = CNN(height=32, width=32, channels=3, class_count=10) needs changing? 
- [ ] learning rate = 0.0002 for us
- [ ] class CNN everything (w.r.t architecture
- [ ] forward function everything change (w.r.t architecture)
- [ ] validate and compute accuracy unsure about for now w.r.t changes made
